styleguide/src/Forms/InputMask/utils.ts

```diff
@@ -32,5 +33,26 @@ export const formatValuePatterns = {
     pipe: maskDate('dd/mm/yyyy'),
     keepCharPositions: true,
     inputMode: 'numeric',
+    placeholder: '__/__/____',
+  },
+  zipCode: {
+    mask: [/\d/, /\d/, /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/],
+    keepCharPositions: true,
+    inputMode: 'numeric',
```

Comment:
Sugiro usar o `decimal`.
Ele fica um pouco melhor no iPhone.
https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/#aa-decimal

Reply:
eu sempre usei decimal, mas vi que o `date` tava em `numeric`, acabei indo igual kk Boa, Roso

Reply:
Ah, mas para o date acho que o numeric é melhor kkkkkk Porque ele tem a barra `/`, acho que pode dar conflito com o decimal